app/api/v1/endpoints/auth.py

```
@@ -485,54 +486,151 @@ async def refresh_token(
+                
+            except JWTError as jwt_error:
+                # Token is invalid/expired - still allow logout
+                logger.info("Logout with invalid/expired token - proceeding anyway", error=str(jwt_error))
+                # Try to extract user ID from token without validation (for cleanup)
+                try:
+                    import jwt as raw_jwt
+                    unverified = raw_jwt.decode(token, options={"verify_signature": False})
+                    user_id = unverified.get("sub")
+                    logger.debug("Extracted user ID from invalid token for cleanup", user_id=user_id)
+                except Exception:
+                    pass  # Can't extract user ID, proceed without it
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)